common.gypi

```diff
@@ -246,6 +247,13 @@
       4838,  # (atlgdi.h) conversion from 'int' to 'UINT' requires a narrowing conversion
       4996,  # (atlapp.h) 'GetVersionExW': was declared deprecated
     ],
+    'conditions': [
+      ['enable_osr==1', {
```

Comment:
The `common.gypi` is a place to put settings for dependencies like Node and breakpad, for configurations of Electron itself, it is probably better to put it in `electron.gyp`.